### PR TITLE
Products Table is created

### PR DESCRIPTION
**Pull request by AyswaryaaR**
Grocery react application base table named "products" is created.

### CODE REVIEW

File: src/App.tsx
Comment (srinathnms1) [inline src/App.tsx:1]: Format all the documents

File: src/interfaces/IProduct.ts
Comment (srinathnms1) [inline src/interfaces/IProduct.ts:1]: Use PascalCase for field names

File: src/products/Products.tsx
Comment (srinathnms1) [inline src/products/Products.tsx:1]: Remove the commented lines and clean up the code

File: src/App.tsx
Comment (AyswaryaaR) [reply src/App.tsx:1]: All documents has been formatted

File: src/interfaces/IProduct.ts
Comment (AyswaryaaR) [reply src/interfaces/IProduct.ts:1]: Pascal case has been applied for all field names

File: src/products/Products.tsx
Comment (AyswaryaaR) [reply src/products/Products.tsx:1]: Commented lines has been removed

File: src/products/AddProduct.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const AddProduct = (props: any) => {
```
Comment (srinathnms1) [inline src/products/AddProduct.tsx]: Create interface for AddProduct props. Avoid “any”

File: src/products/Products.tsx [hunk 2]
```diff
@@ -19,7 +12,7 @@ const Products = (props: IProductProps) => {
               <strong>S.NO</strong>
             </td>
             <td>
-              <strong>Product</strong>
+              <strong>Product</strong>{" "}
             </td>
```
Comment (srinathnms1) [inline src/products/Products.tsx:16]: Remove unwanted space

File: src/products/AddProductDialogBox.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        />
+      </div>
+
+      <div>
+        <input
+          name="Shop name"
+          onChange={(event) =>
+            setProduct({
+              ...product,
+              ShopName: String(event.target.value),
```
Comment (srinathnms1) [inline src/products/AddProductDialogBox.tsx]: Target value is already a string

File: src/products/AddProduct.tsx (hunk 1 above)
Comment (AyswaryaaR) [reply src/products/AddProduct.tsx]: IAddProduct interface has been created in order to avoid any.

File: src/products/Products.tsx (hunk 2 above)
Comment (AyswaryaaR) [reply src/products/Products.tsx:16]: Unwanted space has been removed

File: src/products/AddProductDialogBox.tsx (hunk 3 above)
Comment (AyswaryaaR) [reply src/products/AddProductDialogBox.tsx]: String has been removed in input tags of Name and Shop name as they are already a string

**Review by srinathnms1 — APPROVED** [review]

**Pull request merged**